### PR TITLE
Enforce LOW_S after Nov 13

### PR DESCRIPTION
**Pull request by ptschip**

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -1143,6 +1143,23 @@ std::string FormatStateMessage(const CValidationState &state)
...
+{
+    if (pindexPrev == nullptr)
+    {
+        return false;
+    }
+
+    return IsCashHFEnabled(config, pindexPrev->GetMedianTimePast());
+}
+#endif
+
```
Comment (gandrewstone) [inline src/main.cpp]: Can we get started with isolation by putting these functions in a bitcoincash.cpp and .h file?

File: src/script/interpreter.cpp [hunk 2]
```diff
@@ -982,8 +988,14 @@ bool EvalScript(vector<vector<unsigned char> >& stack, const CScript& script, un
                     }
 
                     // Clean up stack of actual arguments
-                    while (i-- > 1)
+                    while (i-- > 1) {
```
Comment (gandrewstone) [inline src/script/interpreter.cpp]: formatting { needs to be on next line

**Comment by gandrewstone** [comment]
So the LOW_S stuff already existed and all that was needed is to turn it on at the right moment?

**Comment by ptschip** [comment]
it was already a mempool policy...but it wasn't being enforced if blocks
would include it.

File: src/script/interpreter.cpp (hunk 2 above)
Comment (ptschip) [reply src/script/interpreter.cpp]: done...seems that this file never made it into the list of clang formatted files.  I guess you know that, we need a final round of formats for all the subdir's /script /wallet /rpc etc...

File: src/main.cpp (hunk 1 above)
Comment (ptschip) [reply src/main.cpp]: done.

**Pull request merged**